libs/input-mapping/tests/Functional/DownloadFilesAdaptiveBranchTest.php

```diff
@@ -64,7 +65,8 @@ public function testReadFilesAdaptiveWithBranch(): void
             ],
         ];
 
-        $testLogger = new TestLogger();
+        $testHandler = new TestHandler();
+        $testLogger = new Logger('testLogger', [$testHandler]);
```

Comment:
proč to neuděláš v AbstractTestCase  ?

Reply:
protože jsem to nezkoumal jestli to můžu někam přesunout... jen jsem prostě TestLogger nahradil ve všech testech

Reply:
ale klidně to tam přesunu, to je asi to nejmenší